frontend/model/colors.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  'blue': {
+    'name': 'blue',
+    'theme': 'light',
+    'primary': '#5dc8f0',
+    'primarySaturated': '#5dc8f0',
+    'primaryText': '#17b0ea',
+    'primaryTextHover': '#fff',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'green': {
+    'name': 'green',
+    'theme': 'light',
+    'primary': '#c2d879',
+    'primarySaturated': '#90BD0D',
+    'primaryText': '#ffffff',
+    'primaryTextHover': '#363636',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'orange': {
+    'name': 'orange',
+    'theme': 'light',
+    'primary': '#f8b85d',
+    'primarySaturated': '#F89302',
+    'primaryText': '#17b0ea',
+    'primaryTextHover': '#fff',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'silver': {
+    'name': 'silver',
+    'theme': 'light',
+    'primary': '#adadad',
+    'primarySaturated': '#F89302',
+    'primaryText': '#17b0ea',
+    'primaryTextHover': '#fff',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'dark': {
```

Comment:
Can this be renamed this to "black & white"? The "dark" theme content should be white on black per @mmbotelho's [comment here](https://okturtles.slack.com/archives/C0EH7P20Y/p1551277466000900). Also, would be cool if that could be added here too! 😄  If you'd prefer to add it in a separate PR, please open an issue for that so we can keep track of it.

Reply:
Color theme changed